node-js-basics-101/node_modules/ws/lib/WebSocketServer.js

```diff
@@ -0,0 +1,336 @@
+/*!
+ * ws: a node.js websocket client
+ * Copyright(c) 2011 Einar Otto Stangvik <einaros@gmail.com>
+ * MIT Licensed
+ */
+
+'use strict';
+
+const safeBuffer = require('safe-buffer');
+const EventEmitter = require('events');
+const crypto = require('crypto');
+const Ultron = require('ultron');
+const http = require('http');
+const url = require('url');
+
+const PerMessageDeflate = require('./PerMessageDeflate');
+const Extensions = require('./Extensions');
```

Comment:
__[jshint]__
* 'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

<!-- Comment by [SideCI](https://sideci.com) -->